data/Collette_2021/metadata.yml

```diff
@@ -0,0 +1,348 @@
+  NSW:
+    locality: eastern New South Wales
+    description: eastern New South Wales
+    latitude (deg): .na
+    longitude (deg): .na
+    rainfall description: aseasonal rainfall
+  TAS:
+    locality: north-east coast of Tasmania
+    description: north-east coast of Tasmania
+    latitude (deg): .na
+    longitude (deg): .na
+    rainfall description: aseasonal rainfall
+contexts: .na
+traits:
+- var_in: Dormancy
+  unit_in: .na
+  trait_name: seed_dormancy_class
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: We classified dormancy into one of five types, ND, PY, PD, MPD or MD. Dormancy
+    class was mainly assigned to species from information obtained at the genus level
+    from published studies, although we used species-level information when available.
+    Shrub species in temperate fire-prone Australia that are serotinous (i.e. have
+    a canopy seed bank) have non-dormant seeds (e.g. Lamont and Groom, 1998) while
+    the majority of others have soil-stored seeds that are dormant (Auld et al., 2000;
+    Ooi, 2007). The literature was searched using the Google scholar engine with the
+    following parameters, genus name (for each of the genera), seed germination and
+    seed dormancy. Where there were multiple published studies across multiple species
+    within a genus that either confirmed dormancy type or provided evidence of dormancy
+    type (e.g. a permeable seed displaying a smoke response indicative of PD), the
+    genus was included. For example, the well-studied serotinous species in the family
+    Proteaceae (including the genera Banksia, Dryandra, Hakea and Petrophile) have
+    non-dormant seeds held in protective cones (e.g. Lamont and Groom, 1998; Bell,
+    1999; Ooi, 2007) and were confidently assigned to the ND class. Similarly well
+    studied are temperate-climate species with PY across multiple genera (including
+    Acacia, Dillwynia, Dodonaea and Pultenaea) in the Fabaceae, Malvaceae and Sapindaceae
+    (e.g. Auld and O'Connell, 1991; Turner et al 2005; Ooi et al., 2009, 2014) and
+    species with PD or MPD in the families Ericaceae, Rutaceae and Lamiaceae (Ooi,
+    2007; Ooi et al., 2014; Mackenzie et al., 2016) (including Boronia and Phebalium).
+    For genera that were poorly represented in the literature, seed dormancy class
+    was determined by a combination of author knowledge, seed morphological characteristics
+    and information from germination trials conducted by Botanic Gardens' researchers
+    when viability testing seed collections. It was not possible to assign dormancy
+    class confidently to a species level across the full dataset, due to a lack of
+    available literature on every species. As such, some genera that were subject
+    to conflicting reports, where we were not confident in assigning dormancy class,
+    were excluded from the analysis.
+substitutions:
+- trait_name: seed_dormancy_class
+  find: ND
```

Comment:
I didn't know that capitalisation doesn't matter for substitutions! The data is all lowercase.

Reply:
yes - everything turns lowercase for matching.